app/services/server_access/heroku.rb

```diff
@@ -63,6 +63,10 @@ def setup_db
       safely_with_log { execute_command("bundle exec rails db:schema:load", "DISABLE_DATABASE_ENVIRONMENT_CHECK" => 1, "SAFETY_ASSURED" => 1) }
     end
 
+    def run_command(command)
+      safely_with_log { execute_command(command, "SAFETY_ASSURED" => 1) }
```

Comment:
Maybe we must check that command matches `... rails db: ...` or `... rake db: ...`

Reply:
Why?

Reply:
discussed